andino_gazebo/launch/andino_one_robot.launch.py

```diff
@@ -56,9 +56,15 @@ def generate_launch_description():
         default_value='true',
         description='Use simulation (Gazebo) clock if true',
     )
+
+    # Set the path to the world file
+    world_file_name = 'empty.world'
+
```

Comment:
```suggestion
```

Reply:
To save vertical space, you can simply add the name of the file where it needs to be defined. It is not used anywhere else.

Reply:
Fixed!